File: 240328/순서를 바꾸었을 때 같은 단어인지 판별하기/determine-same-word.java
import java.util.Scanner;
import java.util.Arrays;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        String str1 = sc.nextLine();
        String str2 = sc.nextLine();
        char[] ch1 = str1.toCharArray();
        char[] ch2 = str2.toCharArray();
        Arrays.sort(ch1);
        Arrays.sort(ch2);
        int k = 0;
        for(int i=0;i<str1.length();i++) {
            if(ch1[i]!=ch2[i]){
                System.out.println("No");
                break;
            } else {
                k++;
            }
        }
        if(k == str1.length()) {
            System.out.println("Yes");
        }
    }
}

File: 240417/순서를 바꾸었을 때 같은 단어인지 판별하기/determine-same-word.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)

    val str1 = sc.next()
    val str2 = sc.next()

    if (str1.toList().sorted() == str2.toList().sorted()) {
        print("Yes")
    } else {
        print("No")
    }
}